app/controllers/application_controller.rb

```diff
@@ -46,7 +46,7 @@ def after_sign_in_path_for(resource)
   end
 
   def raise_banned
-    raise "BANNED" if current_user&.banned
+    raise "SUSPENDED" if current_user&.banned
```

Comment:
General question: the way this is used is through `before_action`s like this one:

```ruby
before_action :raise_banned, only: %i[new create update]
```

Could it potentially make sense to move this check into a policy?

Reply:
Also on the danger of sounding nitpicky, it's a bit confusing to see a method called `raise_banned` and then `raise "SUSPENDED" right below. Maybe we can rename the method to `raise_suspended`? It only would need to change in 3 places from what I can tell.

Reply:
> Could it potentially make sense to move this check into a policy?

Not sure TBH as I'm totally unfamiliar with all the suspension tooling, but I think it's out of the scope here

> Also on the danger of sounding nitpicky, it's a bit confusing to see a method called raise_banned and then raise "SUSPENDED" right below. Maybe we can rename the method to raise_suspended`? It only would need to change in 3 places from what I can tell.

Done!

Reply:
Agree that a policy would be outside the scope of this PR, but it seemed like a good point to start the discussion. 😃 

Thanks for changing the method name!

Reply:
How painful would it be to rename the role on the user as well? Just looking at this method, we have `current_user&.banned`, which might cause a bit of confusion since "banned" is actually now "suspended". I don't think this should _block_ this PR, but we probably shouldn't have too many different ways to refer to the same concept.

Also, looking at `role.rb`, we also have a `comment_banned` role. Do we need to make any modifications to handle that?

Reply:
Yeah, the goal is to slowly change it all I think. We'd probably need a migration to go over the `roles` table and update that role.

Let's see what we can do in a follow up PR :)

Reply:
That sounds great. Definitely seems like it could be its own PR 😄